sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/AzureMonitorTestEnvironment.cs

```diff
@@ -9,14 +9,18 @@ namespace Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests
 
     public class AzureMonitorTestEnvironment : TestEnvironment
     {
+        public Uri LogsEndpoint => new(GetRecordedVariable("LOGS_ENDPOINT"));
+
         /// <summary>
         /// Connection String is used to connect to an Application Insights resource.
         /// This value comes from the ARM Template.
         /// </summary>
-        public string ConnectionString => GetRecordedVariable("CONNECTION_STRING");
+        public string PrimaryConnectionString => GetRecordedVariable("PRIMARY_CONNECTION_STRING");
```

Comment:
nit: drop primary on the first resource. It may not fit well with existing test cases.